client/my-sites/post-share/style.scss

```diff
@@ -0,0 +1,140 @@
+.posts__list .post-share__wrapper {
```

Comment:
We are doing this pretty soon. Here the whole work-flow: https://github.com/Automattic/wp-calypso/pull/12471